hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/admin/om/GetServiceRolesSubcommand.java

```diff
@@ -30,10 +30,10 @@
 import java.util.concurrent.Callable;
 
 /**
- * Handler of om get-service-roles command.
+ * Handler of om status command.
  */
 @CommandLine.Command(
-    name = "getserviceroles",
+    name = "status",
```

Comment:
I think we should keep `getserviceroles` as an alias for compatibility.

```suggestion
    name = "status", aliases = "getserviceroles",
```

Reply:
Thanks @adoroszlai for the idea, I agree with it.

Reply:
That makes sense. Suggestion applied.